tutorials/luigi-container/luigi-container.md

```diff
@@ -0,0 +1,278 @@
+---
+parser: v2
+auto_validation: true
+time: 20
+tags: [ tutorial>beginner, programming-tool>javascript]
+primary_tag: topic>user-interface
+---
+
+# Implement Micro Frontends in Your UI5 App with Luigi Container
+<!-- description --> Learn how to build a UI5 application that utilizes the Luigi micro frontend framework. With the help of Luigi Container, you can create more modular and reusable applications in a simple way.
+
+## Prerequisites
+ - It is recommended to try the simpler Luigi examples on [GitHub](https://github.com/SAP/luigi/tree/main/core/examples) first to get acquainted with the framework.
+ - You need to have [Node.js](https://nodejs.org/en/download/current/) installed.
+
+## You will learn
+  - The basics of the Luigi micro frontend framework and Luigi Container 
+  - How to install Luigi Container
+  - How to use Luigi Container inside a sample UI5 application
+
+## Intro
+
+[Project "Luigi"](https://luigi-project.io) is an open-source micro frontend framework suitable for SAP environments, providing Fiori-compliant navigation out-of-the-box. Luigi is technology-agnostic, allowing you to create your app using any given frontend toolkit. 
+
+Normally, Luigi consists of two main parts: **Luigi Core** and **Luigi Client**. Core refers to the main app which houses the micro frontends, while Client refers to the micro frontends themselves. 
+
+However, a third feature called [Luigi Container](https://docs.luigi-project.io/docs/luigi-container) allows you to easily insert Luigi micro frontends anywhere without the need for a Luigi Core application. This feature simplifies the use of Luigi for developers, eliminating the need for significant changes in their applications.
+
+This tutorial will show you how to incorporate micro frontends inside a [UI5](https://sdk.openui5.org) application. Nevertheless, a similar process will also apply to other frontend frameworks such as [Angular](https://angular.io/) or [React](https://react.dev/).
+
+> **Note:** In addition to this tutorial, you can also experiment with the Luigi Container test app on [GitHub](https://github.com/SAP/luigi/tree/main/container/test-app).
+
+---
+
+## Steps
```

Comment:
You can remove this row.  It is not necessary.

Reply:
Hi @jmmargo , thank you for the comments! I implemented the changes.